source/sonata_simulation.rst

```diff
@@ -409,7 +409,7 @@ Collection of dictionaries with each member describing one data collection durin
    dt                             float      Mandatory    Interval between reporting steps in milliseconds. If assigned value smaller than simulation dt, will be set equal to simulation dt.
    start_time                     float      Mandatory    Time to start reporting in milliseconds.
    end_time                       float      Mandatory    Time to stop reporting in milliseconds.
-   file_name                      text       Optional     Specify file name, otherwise takes the form <repname>_SONATA.h5. If '.h5' extension is not included, it will be added.
+   file_name                      text       Optional     Specify file name, otherwise takes the form <repname>.h5. If '.h5' extension is not included, it will be added.
```

Comment:
while we're at it, we may aswell clarify `repname`, as it's not mentioned anywhere else.

Reply:
maybe is worth mentioning that the <report_name>.h5 is coming from the key of the dictionary?

Reply:
Sure, I've modified the text again. Does it look better?

Reply:
LGTM!